NoahPBM/contracts/paymentmanager/NoahPaymentManager.sol

```
@@ -370,26 +328,13 @@ contract NoahPaymentManager is
 
         // Ensure that only campaign PBM can spend its own money
         if (pbmTokenBalance[campaignPBM][erc20Token] >= erc20TokenValue) {
```

should change this to require() also to match the above changes.

---

cause i realizd that if its an IF condition, itwould just fail silently, which will affect PBM logic, if no error raise, but the ERC1155 token burnt already.